core/runner/pom.xml

```diff
@@ -102,14 +110,6 @@
       <groupId>com.googlecode.zohhak</groupId>
       <artifactId>zohhak</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-module-junit4</artifactId>
-    </dependency>
   </dependencies>
```

Comment:
We can probably remove also:
```
<dependency>
      <groupId>org.apache.sling</groupId>
      <artifactId>org.apache.sling.commons.osgi</artifactId>
</dependency>
```

As `org.apache.sling.commons.osgi.PropertiesUtil` is no longer used.

Reply:
Done